test/navigation/NavigatorSpec.scala

```diff
@@ -226,28 +231,28 @@ class NavigatorSpec extends SpecBase {
             )
           }
 
-          "to CheckYourAnswers when NiphlNumberPage is answered" in {
+          "to CyaNirmsNiphl when NiphlNumberPage is answered" in {
 
             val answers =
               UserAnswers("id").set(HasNiphlPage, true).success.value.set(NiphlNumberPage, "1234").success.value
-            navigator.nextPage(HasNiphlPage, CheckMode, answers) mustBe routes.CheckYourAnswersController.onPageLoad
+            navigator.nextPage(HasNiphlPage, CheckMode, answers) mustBe routes.CyaNirmsNiphlController.onPageLoad
           }
         }
 
-        "to CheckYourAnswersPage when answer is No" in {
+        "to CyaNirmsNiphl when answer is No" in {
```

Comment:
CyaNirmsNiphl - is this what we've named the profile set up CYA controller?

Reply:
Yes :)

Reply:
Changed name